tests/conftest.py

```diff
@@ -67,7 +67,7 @@ def containers(request, confdir):
     if request.config.option.usedocker:
         docker = request.getfixturevalue('dockerctl')
         with docker.run(
-            'safarov/freeswitch',
+            'safarov/freeswitch:1.10.3',
```

Comment:
We should probably have a CD process eventually that keeps us in sync with core fs.

Reply:
Looks like the maintainer added a latest tag 4 months ago. I'll use that tag instead.